src/GameState.h

```diff
@@ -68,8 +68,9 @@ class GameState : public KoState {
     void start_clock(int color);
     void stop_clock(int color);
     const TimeControl& get_timecontrol() const;
+    void set_timecontrol(const TimeControl& timecontrol);
     void set_timecontrol(int maintime, int byotime, int byostones,
-                         int byoperiods);
+        int byoperiods);
```

Comment:
Whitespace change. Also I think most of Leela's code aligns arguments whenever possible.